src/components/POLine/POLineForm.js

```diff
@@ -72,7 +72,7 @@ class POLineForm extends Component {
         [ACCORDION_ID.itemDetails]: false,
         [ACCORDION_ID.physical]: false,
         [ACCORDION_ID.fundDistribution]: false,
-        [ACCORDION_ID.location]: false,
+        [ACCORDION_ID.location]: true,
```

Comment:
I don't think this should be the case - always opened accordion with Locations.

Reply:
but locations became required field and we should provide at least a one for setting